driver-core/src/main/java/com/datastax/driver/core/DataTypeCqlNameParser.java

```diff
@@ -148,7 +148,7 @@ static DataType parse(String toParse, Cluster cluster, String currentKeyspaceNam
         if (userType == null)
             throw new UnresolvedUserTypeException(currentKeyspaceName, type);
         else
-            return userType;
+            return userType.copy(frozen);
```

Comment:
Makes sense, but for my understanding was wondering if there would ever be a case where the existing type's `frozen` flag doesn't match the input `frozen` flag